[2025-04-01T18:43:35.702+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-01T18:43:35.711+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.task2 manual__2025-04-01T18:43:31.529210+00:00 [queued]>
[2025-04-01T18:43:35.715+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.task2 manual__2025-04-01T18:43:31.529210+00:00 [queued]>
[2025-04-01T18:43:35.715+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-04-01T18:43:35.720+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): task2> on 2025-04-01 18:43:31.529210+00:00
[2025-04-01T18:43:35.726+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=5129) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-01T18:43:35.727+0000] {standard_task_runner.py:72} INFO - Started process 5130 to run task
[2025-04-01T18:43:35.725+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'task2', 'manual__2025-04-01T18:43:31.529210+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpcl7uct0b']
[2025-04-01T18:43:35.730+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask task2
[2025-04-01T18:43:35.806+0000] {task_command.py:467} INFO - Running <TaskInstance: first_dag.task2 manual__2025-04-01T18:43:31.529210+00:00 [running]> on host 7d980c7fabe2
[2025-04-01T18:43:36.408+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='grypton' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T18:43:31.529210+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-01T18:43:31.529210+00:00'
[2025-04-01T18:43:36.442+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-01T18:43:36.462+0000] {logging_mixin.py:190} INFO - Hello from task2 PythonOperator
[2025-04-01T18:43:36.467+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-01T18:43:36.549+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-01T18:43:36.557+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=task2, run_id=manual__2025-04-01T18:43:31.529210+00:00, execution_date=20250401T184331, start_date=20250401T184335, end_date=20250401T184336
[2025-04-01T18:43:36.878+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-01T18:43:36.902+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-01T18:43:36.904+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
